.github/workflows/test-model.yml

```diff
@@ -0,0 +1,59 @@
+name: test-model
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: null
```

Comment:
```suggestion
  pull_request:
```

Reply:
Changing to workflow_dispatch / cron (again, this is a leftover from `tests.yml`, my bad)